src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const oneDayPay = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const discountForShortTerm = 20;
+  const discountForLongTerm = 50;
```

Comment:
The same issue here. This constant should be named with uppercase
<img width="665" alt="Screenshot 2023-09-25 at 13 09 39" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/b5396b8d-5189-4c5b-9f94-0ad9536a0e9a">
https://medium.com/@kjschelling/a-comprehensive-guide-to-naming-conventions-in-javascript-46a7518e4807

Reply:
Ok, upper case for variables made